### PR TITLE
add potsdam file

### PR DESCRIPTION
**Pull request by kkrebs**
adding the file that names places to eat in Potsdam, Germany

### CODE REVIEW

**Comment by yagee-de** [comment]
Is there nothing else in Potsdam?

**Comment by yagee-de** [comment]
Forget to use @kkrebs

File: kkrebs-potsdam.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Potsdam, Germany
+
```
Comment (yagee-de) [inline kkrebs-potsdam.md:2]: No places to visit in this town?

**Review by yagee-de — CHANGES_REQUESTED** [review]
Add some local spots

File: kkrebs-potsdam.md (hunk 1 above)
Comment (kkrebs) [reply kkrebs-potsdam.md:2]: @yagee-de yes, but they are full of tourists

**Comment by yagee-de** [comment]
you merged it on your own? 😢

**Comment by kkrebs** [comment]
@yagee-de ... and now reverted it

**Pull request merged**